dev-python/spyder-notebook/spyder-notebook-0.2.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Jupyter notebook integration with Spyder "
+HOMEPAGE="https://github.com/spyder-ide/spyder-notebook"
+SRC_URI="https://github.com/spyder-ide/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+IUSE="doc"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/nbformat[${PYTHON_USEDEP}]
+	dev-python/notebook[${PYTHON_USEDEP}]
+	dev-python/psutil[${PYTHON_USEDEP}]
+	dev-python/QtPy[webkit,${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	>=dev-python/spyder-4.0.0[${PYTHON_USEDEP}]"
+
+# Tests do not work inside virtx/emerge for some reason, core dumped
+
+python_install_all() {
+	use doc && dodoc -r doc/*
```

Comment:
Depending on the size of this, I'd say you can just install them without having a USE flag here. You can use a global DOCS array and then it gets installed with `distutils-r1_python_install_all` and you can remove `python_install_all()` function from the ebuild.